lib/main.js

```diff
@@ -286,7 +286,7 @@ setReadOnly( stdlib, 'random', require( '@stdlib/random' ) );
 * @type {Namespace}
 * @see {@link module:@stdlib/regexp}
 */
-setReadOnly( stdlib, 'regexp', require( '@stdlib/regexp' ) );
+setReadOnly( stdlib, 'RegExp', require( '@stdlib/regexp' ) );
```

Comment:
```suggestion
setReadOnly( stdlib, 'regexp', require( '@stdlib/regexp' ) );
```

Reverting this as `regexp` returns to the namespace object, not the constructor.